### ISSUE TITLE
Bad results with pre-trained blank models

### ISSUE THREAD

**Issue by TANGTANG-BOY** [plkmo/BERT-Relation-Extraction#49]
Hello, I have recently been using this program of yours for learning, but I have noticed that when I use the blank pre-trained model for task fine-tuning I get poor results, not up to what is shown in your results, I haven't changed any of the code, but it seems like there is overfitting when I change the learning rate, the results are as follows:

![Image](https://github.com/user-attachments/assets/adb9aafc-01de-4d77-ae48-4be0414b67b6)

![Image](https://github.com/user-attachments/assets/08ee37e9-aa04-4194-b4b2-ef0394f66a0d)
